scripts/c2r_script.py

```diff
@@ -8,6 +8,12 @@
 
 from urllib2 import urlopen
 
+# SCRIPT_TYPE is either 'CONVERSION' or 'ANALYSIS'
+# Value is set in signed yaml envelope in content_vars (CONVERT2RHEL_SCRIPT_TYPE)
+SCRIPT_TYPE = os.getenv("RHC_WORKER_CONVERT2RHEL_SCRIPT_TYPE", None)
```

Comment:
We will need to rename the env var here... anything that starts with `CONVERT2RHEL_*` in its name will be collected by convert2rhel. 

That type of information would be duplicated and maybe not needed.

Reply:
Hmm, I changed it to simple `RHC_WORKER_SCRIPT_MODE`, `_TYPE` indicated more like a python or bash type, `_MODE` sounds better. WDYT?